packages/libcurl.rb

```diff
@@ -72,6 +66,6 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-    FileUtils.rm "#{CREW_DEST_PREFIX}/bin/curl"
+    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/curl", "#{CREW_DEST_PREFIX}/bin/curl.nonstatic"
```

Comment:
The only binary generated by this package is `/usr/local/bin/curl-config` so we remove this line.

Reply:
Honestly I was leaving it there as a curl failsafe...